.gitignore

```diff
@@ -1,6 +1,80 @@
+# OS files
 .DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+ehthumbs.db
+Thumbs.db
 themes/.DS_Store
-.hugo_build.lock
+
+# Dependencies
+node_modules/
+**/node_modules/
+package-lock.json
+yarn.lock
+**/yarn.lock
+pnpm-lock.yaml
```

Comment:
I don't agree on ignoring lock files, since this is a common practice we do in order to make a deterministic build process, especially for CI. 

I know that a proper dependency configuration on package.json would not need any lock file, but sometimes there are unintended issues injected by the dependencies (e.g. a breaking change in a new patch version) and it becomes difficult to debug (why does it work in my computer and not yours?)